src/Dialog/Dialog.js

```diff
@@ -141,7 +141,7 @@ function getStyles(props, context) {
       fontSize: dialog.bodyFontSize,
       color: dialog.bodyColor,
       padding: gutter,
-      paddingTop: props.title ? 0 : gutter,
+      paddingTop: 0,
```

Comment:
@oliviertassinari FYI, I have a version of dialog locally that does not require any `padding` on the dialog container at all. All the padding is on subcomponents `DialogTitle`, `DialogContent`